### PR TITLE
docs: expand command documentation

### PR DESCRIPTION
**Pull request by ComradeVanti**
This PR expands the command documentation by giving each command or topic it's own file where detailed information can be collected. The relevant file can also be more easily linked.

This doc change also considered what is discussed in #408.

### CODE REVIEW

**Comment by ComradeVanti** [comment]
https://github.com/openupm/openupm-cli/discussions/408#discussioncomment-10819184

Just linking this here, so I dont forget.

**Comment by ComradeVanti** [comment]
@favoyang you want to proofread me one more time? thanks :)

File: docs/help-package-reference.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The specific given version.
+
+Example: `com.my.package@1.2.0`
+
+## The stable tag
+
+This resolves the latest published stable version, excluding pre-releases
+
+Example: `com.my.package@latest`
```
Comment (favoyang) [inline docs/help-package-reference.md:31]: This should be `com.my.package@stable`

**Review by favoyang — CHANGES_REQUESTED** [review]

File: docs/help-package-reference.md (hunk 1 above)
Comment (ComradeVanti) [reply docs/help-package-reference.md]: true. i fixed it

**Pull request merged**